README.md

```diff
@@ -76,21 +113,37 @@ You can observe that:
   kubectl get pods -n liqo-demo -o wide  
 ``` 
 
-To get more information about how to install Liqo in your own clusters and configure it, you can check out the [Liqo Documentation](https://doc.liqo.io/user/).
+## Installation
+
+Liqo can be installed by using Helm. The configuration depends on the cluster type (K8s, K3s, managed) and the provider. Possible scenarios can be found [here](https://doc.liqo.io/user/scenarios/).
 
-## Architecture
+To be completed
 
-Liqo relies on several components:
+## Roadmap
 
-* *Liqo Virtual Kubelet*: Based on [Virtual Kubelet](https://github.com/virtual-kubelet/virtual-kubelet) project, the VK
- is responsible to "masquerade" a foreign Kubernetes cluster.
-* *Advertisement Operator/Broadcaster*: Those components embed the logic to advertise/accept resources from partner
- clusters and spawn new virtual kubelet instances
-* *Liqonet Operators*: Those operators are responsible to establish Pod-to-Pod and Pod-to-Service connection across 
-partner clusters.
+In the next releases, Liqo plans to have implemented:
 
-...and some others. Check out the architecture [documentation](https://doc.liqo.io/architecture/)
+* Support to more than one cluster to join with
+* Support to Amazon Elastic Kubernetes Service
+* Support to Decentralized governance
```

Comment:
"Support to Decentralized governance" ==> I thought it was already part of the project.
Be more specific.